src/app/execution.rs

```diff
@@ -122,35 +126,100 @@ impl ExecutionContext {
         &self.session_ctx
     }
 
+    pub async fn run_sqls(&self, sqls: Vec<&str>, sender: UnboundedSender<AppEvent>) -> Result<()> {
```

Comment:
I eventually hope to have a stronger API between the execution context and the App (aka have it so the ExecutionContext can be used directly by people without also the UI / CLI). Can be done in the future too though

Reply:
I hope to take a stab at this today